tests/environment/docker-compose-mtls.yml

```diff
@@ -0,0 +1,299 @@
+# This docker-compose file will spin up an ACS cluster on a local host.
+# It requires a minimum of 13GB Memory to distribute among containers.
+#
+# For performance tuning, assign the container memory and give a percentage of
+# it to the JVM. Use either the -Xms,-Xmx flags or the newly added flags in
+# java 10+: -XX:MaxRAMPercentage and -XX:MinRAMPercentage.
+# More details here:
+# https://www.oracle.com/technetwork/java/javase/10-relnote-issues-4108729.html
+#
+# Note: The docker-compose file from github.com is a limited trial that goes
+# into read-only mode after 2 days. Get the latest docker-compose.yml file with
+# a 30-day trial license by accessing the Alfresco Content Services trial
+# download page at:
+# https://www.alfresco.com/platform/content-services-ecm/trial/download
+#
+# Using version 2 as 3 does not support resource constraint options
+# (cpu_*, mem_* limits) for non swarm mode in Compose
+---
+version: "2"
+services:
+  alfresco:
+    image: alfresco/alfresco-content-repository:latest
+    mem_limit: 1900m
+    environment:
+      CATALINA_OPTS : "-agentlib:jdwp=transport=dt_socket,address=*:8000,server=y,suspend=n"
+      JAVA_TOOL_OPTIONS: >-
+        -Dencryption.keystore.type=JCEKS
+        -Dencryption.cipherAlgorithm=DESede/CBC/PKCS5Padding
+        -Dencryption.keyAlgorithm=DESede
+        -Dencryption.keystore.location=/usr/local/tomcat/shared/classes/alfresco/extension/keystore/keystore
+        -Dmetadata-keystore.password=mp6yc0UD9e
+        -Dmetadata-keystore.aliases=metadata
+        -Dmetadata-keystore.metadata.password=oKIWzVdEdA
+        -Dmetadata-keystore.metadata.algorithm=DESede
+      JAVA_OPTS: >-
+        -Dalfresco.port=8082
+        -Dalfresco.host=localhost
+        -Daos.baseUrlOverwrite=http://localhost:8082/alfresco/aos
+        -Ddb.driver=org.postgresql.Driver
+        -Ddb.username=alfresco
+        -Ddb.password=alfresco
+        -Ddb.url=jdbc:postgresql://postgres:5432/alfresco
+        -Dsolr.host=solr6
+        -Dsolr.port=8983
+        -Dsolr.secureComms=secret
+        -Dsolr.sharedSecret=secret
+        -Dsolr.base.url=/solr
+        -Dindex.subsystem.name=solr6
+        -Dalfresco.restApi.basicAuthScheme=true
+        -Dtransform.service.enabled=false
```

Comment:
This property is defined twice, also, if possible, please group the properties so they aren't in a scattered order :) (sorry :D)